docscan/sandbox/request/task/document_text_data_extraction_task_test.go

```diff
@@ -14,7 +14,7 @@ func ExampleDocumentTextDataExtractionTaskBuilder() {
 		return
 	}
 
-	check, err := NewDocumentTextDataExtractionTaskBuilder().
+	task, err := NewDocumentTextDataExtractionTaskBuilder().
```

Comment:
Some linters complain if variable names are the same as package names in the project. But since we are within the `task` package, I think this is fine? So e.g. having `filter` here is not recommended, since we import `"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/filter"`